shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -659,14 +677,32 @@
             break;
 
           case 'touchstart':
-            this.canceled = e.touches.length > 1;
-            break;
+            this.cancelLongPressTimeout();
 
-          case 'contextmenu':
-            if (this.icon || this.canceled) {
+            if (e.touches.length > 1) {
+              this.canceled = true;
               return;
             }
 
+            if (this.inEditMode) {
+              this.touchStart.x = e.touches[0].pageX;
```

Comment:
As it only gets used once, this doesn't really save any space - touchStart ends up being that reference. Will leave this as is I think.